css-scoping-1/Overview.bs

```diff
@@ -424,6 +424,44 @@ Selecting Slot-Assigned Content: the ''::slotted()'' pseudo-element</h4>
 	The only way to style assigned text nodes
 	is by styling the <a>slot</a> and relying on inheritance.
 
+<h4 id='has-slotted-pseudo'>
+Matching Selectors Against Slot-Assigned Content: the '':has-slotted()'' pseudo-class</h4>
+
+	The <dfn selector>:has-slotted()</dfn> pseudo-class, when evaluated 
+	<a>in the context of a shadow tree</a>,
+	matches the <a>slot</a>, if one of its
+	<a lt="find flattened slottables">assigned, after flattening,</a> matches the provided <<compound-selector>>.
```

Comment:
"if one of its assigned, after flattening, matches "

Is there a missing work there? "child elements"?